extensions/geode-modules-test/src/main/java/org/apache/geode/modules/session/catalina/DeltaSessionJUnitTest.java

```diff
@@ -195,6 +199,65 @@ public void serializeReturnsSerializedObject() throws IOException {
     assertThat(result).isEqualTo(serializedObj);
   }
 
+  @Test
+  public void setDeserializedAttributesValueInvokesGetAttribute() {
+    when(manager.getPreferDeserializedForm()).thenReturn(true);
+    DeltaSession session = spy(new DeltaSession(manager));
+    String attributeName = "name";
+    List<String> attributeList = new ArrayList<>();
+    attributeList.add(attributeName);
```

Comment:
This can be replaced with:
`Collection<String> attributeList = Collections.singelton(attributeName);`